app/models/katello/candlepin/content.rb

```diff
@@ -51,28 +51,40 @@ def find_repository
       end
 
       def build_repository
-        ::Katello::Repository.new(:environment => product.organization.library,
-                                 :product => product,
-                                 :pulp_id => pulp_id,
-                                 :cp_label => content.label,
-                                 :content_id => content.id,
-                                 :arch => arch,
-                                 :major => major,
-                                 :minor => minor,
-                                 :relative_path => relative_path,
-                                 :name => name,
-                                 :label => label,
-                                 :url => feed_url,
-                                 :feed_ca => ca,
-                                 :feed_cert => product.certificate,
-                                 :feed_key => product.key,
-                                 :content_type => katello_content_type,
-                                 :preserve_metadata => true, #preserve repo metadata when importing from cp
-                                 :unprotected => unprotected?,
-                                 :content_view_version => product.organization.library.default_content_view_version)
+        repository = Repository.new(
+          :environment => product.organization.library,
+          :product => product,
+          :pulp_id => pulp_id,
+          :cp_label => content.label,
+          :content_id => content.id,
+          :arch => arch,
+          :major => major,
+          :minor => minor,
+          :relative_path => relative_path,
+          :name => name,
+          :label => label,
+          :url => feed_url,
+          :feed_ca => ca,
+          :feed_cert => product.certificate,
+          :feed_key => product.key,
+          :content_type => katello_content_type,
+          :preserve_metadata => true, #preserve repo metadata when importing from cp
+          :unprotected => unprotected?,
+          :content_view_version => product.organization.
+                                  library.default_content_view_version
+        )
+
+        if katello_content_type == Repository::OSTREE_TYPE
+          # TODO
+          # add a hard coded config entry branch for now
+          # we will add pull all once the summary files functionality is available
+          repository.ostree_branches.new(:name => Setting[:default_cdn_ostree_branch_name], :repository => repository)
```

Comment:
Saw this comment and wanted to ask about it given this was written a while back and Pulp 2.8 has summary support.